src/components/ItemCard.jsx

```diff
@@ -42,21 +46,75 @@ function ItemCard (props) {
     setEditOpen(false)
   }
 
+  function handleMarkItemReturned () {
+    const item = {
```

Comment:
this isn't super obvious (bad naming again on my part), but `UpdateItemAsync` shouldn't need to be passed every item attribute. it should work if you just pass the status.

Reply:
I just tried it without, and it breaks on the back end (it's not a 401, make sure). We do something similar for the edit card modal, so I think this can stay as is.

Edit card modal thunk:
```
    dispatch(updateItemAsync({
      id: itemId,
      name,
      type,
      location,
      description
    }))
```

Reply:
ah, that's probably because i should have had `updateItemAsync` take the id as a parameter instead of taking it from the object, e.g. `updateItemAsync(itemId, { status: STATUS.AVAILABLE })`. maybe i'll change that at some point.